src/frontend/src/routes/(new-styling)/authorize/(authenticated)/upgrade-success/+page.svelte

```diff
@@ -21,5 +41,8 @@
       fingerprint, face or screen lock instead.
     </p>
   </div>
-  <Button variant="primary" onclick={handleRedirect}>Go to app</Button>
+  <Button variant="primary" onclick={handleRedirect}>
+    Go to the app {#if countdown > 0}
+      - ({countdown}){/if}
```

Comment:
By default numbers aren't the same width, resulting in the button width jumping around, but you can tell the font to render numbers with the same width.

```suggestion
      - (<span class="tabular-nums">{countdown}</span>){/if}
```

Reply:
Nice!